lesson-6/public/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+  Name:<input type="text" id="myName" />
+  Comment:<input type="text" id="myComment" />
+  <input type="submit" id="button" />
+<div>
+  <pre id="chat"/>
+  </div>
+  <script>
+const button =document.getElementById("button");
+button.onclick=()=>{
+  const name=document.getElementById("myName").value;
```

Comment:
I would change the `name` variable to something else since `name` is actually a native name.